### ISSUE TITLE
Warning about GLAPIENTRY macro redefined

### ISSUE THREAD

**Issue by Jacalz** [glfw/glfw#2010]
Compiling for BSD (have seen it on FreeBSD, NetBSD and I think OpenBSD as well) seems to print a warning about the GLAPIENTRY macro being redefined.  I know that this is only a warning, but it would be great if this could be fixed/silenced so that we don't have to see it when compiling for those platforms. Compiling for Linux and Windows is free from warning in my testing.

```
In file included from /go/pkg/mod/github.com/go-gl/glfw/v3.3/glfw@v0.0.0-20211024062804-40e447a793be/native_linbsd.go:10:
In file included from /go/pkg/mod/github.com/go-gl/glfw/v3.3/glfw@v0.0.0-20211024062804-40e447a793be/glfw/include/GLFW/glfw3native.h:114:
In file included from /freebsd/usr/local/include/GL/glx.h:32:
/freebsd/usr/local/include/GL/gl.h:56:11: warning: 'GLAPIENTRY' macro redefined [-Wmacro-redefined]
/go/pkg/mod/github.com/go-gl/glfw/v3.3/glfw@v0.0.0-20211024062804-40e447a793be/glfw/include/GLFW/glfw3.h:5894:10: note: previous definition is here
```

I think it is an issue here at least. Feel free to close if it is not.

**Comment by elmindreda** [comment]
This should be fixed now for all Unices with 535c3ce63240ee91f998219b35dc82abef545c3d. Thank you!

**Comment by Jacalz** [comment]
Wonderful! Thanks for fixing it :)
